bot/message_syntax_functions.py

```diff
@@ -0,0 +1,339 @@
+"""Module containing functions for converting text inside the Embed Creator
+messages."""
+
+import discord
+
+
+def find_single_member(ctx, member_name: str) -> str:
+    """Takes the string and returns the corresponding member from the discord server.
+
+    Each name on the discord server looks like this: name#XXXX,
+    where "XXXX" is any 4-digit number. The function reads the entire string,
+    finds where the # symbol is, treats it as a dividing line to create
+    two new strings, which are used to look up the formatted member's name
+    from the server's database.
+
```

Comment:
Mógłbyś sprawdzić, czy to jest nadal aktualne z tego co się orientuję to discord właśnie odszedł od 4-cyfrowych identyfikatorów https://discord.com/blog/usernames